server/src/main/java/org/opensearch/index/translog/transfer/TranslogTransferManager.java

```diff
@@ -180,11 +191,17 @@ private void downloadToFS(String fileName, Path location, String primaryTerm) th
         if (Files.exists(filePath)) {
             Files.delete(filePath);
         }
+
+        long bytesToRead;
         try (InputStream inputStream = transferService.downloadBlob(remoteDataTransferPath.add(primaryTerm), fileName)) {
+            // Capture number of bytes for stats before reading
+            bytesToRead = inputStream.available();
```

Comment:
is `inputStream.available()` an in-memory variable access?

Reply:
Yes

Reply:
@ashking94 do you propose any changes needed here?

Reply:
No.